app/src/main/res/layout/new_file_dialog.xml

```diff
@@ -67,7 +67,6 @@
                 android:layout_marginRight="32dp"
                 android:layout_weight="30"
                 android:gravity="end"
-                android:hint=".md"
```

Comment:
This default looks like an extension is present when it isn't. The UI is clearer when it is not present.

Reply:
maybe .extension?

Reply:
I'll test it. Mostly I'm fed up with accidentally creating `foo` when I wanted `foo.md` :)

Reply:
The main issue is - we cant know what the extension is and what is not, otherwise we would have just single line text input

like

```
myfile.md.txt
a.txt.jenc
my.cool.text.txt
my.learning.with.zim.txt
```

because you might or might not have multiple file extenions